.eslintrc.cjs

```diff
@@ -5,6 +5,7 @@ module.exports = {
   env: {
     browser: true,
   },
+  parser: '@typescript-eslint/parser',
```

Comment:
[Nit] I think this might be set by `@viamrobotics/eslint-config`, and may not be needed. However, it is not harmful to include